src/gui/src/Action.h

```diff
@@ -28,6 +29,17 @@ class ActionDialog;
 class QSettings;
 class QTextStream;
 
+namespace SETTINGS {
```

Comment:
This needs to be camel case, and more descriptive e.g. SettingsKeys

Reply:
Planning to use this namespace for all the settings we have around the various Qt object. 
Using `SETTINGS` as this is not a type you can make just really a way to tag some strings like an enum

Reply:
Done `SettingsKeys` is the namespace